.github/PULL_REQUEST_TEMPLATE

```diff
@@ -1,22 +1,41 @@
-## Scope and purpose
+<!--
+## Remove this section
 
-Add a few words about why this PR is needed and what is its scope.
+Have a look at [our developer documentation](https://twistedmatrix.com/trac/wiki/TwistedDevelopment#SubmittingaPatch) before submitting your Pull Request.
 
-Add mentions of things that are not covered here and are planed to be done
-in separate PRs.
+Note that the Trac ticket, news fragment, and review submission portions of this process apply to *all* pull requests, no matter how small;
+if you don't do them, it's likely that nobody will even notice your PR needs a review.
+-->
 
+## Scope and purpose
 
-## Remove this section
+Add a few words about why this PR is needed and what is its scope.
 
-Please have a look at [our developer documentation](https://twistedmatrix.com/trac/wiki/TwistedDevelopment#SubmittingaPatch) before submitting your Pull Request.
+Add any comments about trade-offs (if any) made in this PR and the reasoning behind them.
 
-Please note that the trac ticket, news fragment, and review submission portions of this process apply to *all* pull requests, no matter how small; if you don't do them, it's likely that nobody will even notice your PR needs a review.
+Add mentions of things that are not covered here and are planed to be done
+in separate PRs.
 
 
 ## Contributor Checklist:
 
 * [ ] The associated ticket in Trac is here: https://twistedmatrix.com/trac/ticket/<!-- Create a new one at https://twistedmatrix.com/trac/newticket and replace this comment with the ticket number. -->
-* [ ] I ran `tox -e lint` to format my patch to meet the [Twisted Coding Standard](https://twistedmatrix.com/documents/current/core/development/policy/coding-standard.html)
```

Comment:
Most of us use `pre-commit`, but one-off contributors, who benefit most from this checklist, perhaps do not. You're right that CI catches it, but I don't see the harm in keeping this here.

Reply:
FWIW, I was pretty confused when I pulled down trunk and `tox -e black-reformat` no longer worked. This sort of workflow change should be announced on the mailing list. As the contributor docs weren't updated either I only learned what I was supposed to do from this checklist item.

Reply:
Thanks Tom for the feedback. Agree with the confusion. 

I see twisted dev workflow as a wild wild west. Everyone has it's style.
For example, I don't like tox for local dev and TDD  and don't use `tox -e lint`

So for now, the only gold standard in terms of workflow are the required CI checks.